torch/package/package_exporter.py

```diff
@@ -170,7 +178,7 @@ def __init__(
         self,
         f: Union[str, Path, BinaryIO],
         importer: Union[Importer, Sequence[Importer]] = sys_importer,
-        verbose: bool = True,
+        verbose: int = VerbosityLevel.NONE,
```

Comment:
This is a BC breaking change. We don't commit to BC yet for this API, but it will break internal callers, right? Or is there implicit casting going on.

Reply:
We can go fix all of the uses, right? 

We could also keep `verbose` around just for a little bit to provide BC and then remove it later?

Reply:
True/False will cast to the ints implicitly, it'll just break all the static typing checks. I think fixing them wouldn't be too difficult